app/models/services/service_credential_binding.rb

```diff
@@ -0,0 +1,44 @@
+module VCAP
+  module CloudController
+    module ServiceCredentialBindingViews
+      module Types
+        SERVICE_KEY = 'key'.freeze
+        SERVICE_BINDING = 'app'.freeze
+      end
+
+      SERVICE_KEY = ServiceKey.select(
+        :guid,
+        Sequel.as(Types::SERVICE_KEY, :type),
+        Sequel.as(Sequel.cast(:service_instance_id, String), :service_instance_id)
```

Comment:
This bit was pretty weird, and only cropped up when I ran this against MySQL.  MySQL will actually turn the field name into "CAST service_instance_id AS char" or something similar, so the `as` remaps it back to being `service_instance_id`